resources/js/src/app/components/item/OrderPropertyListItem.vue

```diff
@@ -276,7 +276,17 @@ export default {
         {
             const lastChar = value.slice(-1);
 
-            value = parseFloat(value.replace(App.decimalSeparator, "."));
+            const str = String(value.replace(App.decimalSeparator, "."));
```

Comment:
Wozu der string cast? .replace gibt doch einen String zurück

```suggestion
            const str = value.replace(App.decimalSeparator, ".");
```